docs/docs/action-server/add-sanic-extensions.mdx

```diff
@@ -0,0 +1,56 @@
+---
+id: add-sanic-extensions
+sidebar_label: Adding Sanic Extensions
+title: Add Sanic Extensions
+---
+
+# Add Sanic extensions
+With the use of Pluggy, you can now create additional sanic extensions by accessing the app object created by the action server. The hook implemented provides you access 
```

Comment:
This is an implementation detail, we can leave this out:
```suggestion
You can now create additional Sanic extensions by accessing the app object created by the action server. The hook implemented provides you access 
```

Reply:
Could you please address this first, it's removing the implementation reference to pluggy.